src/plugins/controls/server/options_list/options_list_cluster_settings_route.ts

```diff
@@ -8,18 +8,17 @@
 
 import { getKbnServerError, reportServerError } from '@kbn/kibana-utils-plugin/server';
 import { CoreSetup } from '@kbn/core/server';
-import { errors } from '@elastic/elasticsearch';
 
 export const setupOptionsListClusterSettingsRoute = ({ http }: CoreSetup) => {
   const router = http.createRouter();
   router.get(
     {
-      path: '/api/kibana/controls/optionsList/getClusterSettings',
+      path: '/api/kibana/controls/optionsList/getExpensiveQueriesSetting',
       validate: false,
     },
     async (context, _, response) => {
       try {
-        const esClient = (await context.core).elasticsearch.client.asCurrentUser;
+        const esClient = (await context.core).elasticsearch.client.asInternalUser;
```

Comment:
Can we add some comments about why internal user is being used and why its safe

Reply:
Done!